Makefile

```diff
@@ -292,10 +292,12 @@ ifneq ($(wildcard $(XSMM_DIR)/lib/libxsmm.*),)
   $(libceeds) : LDFLAGS += -L$(XSMM_DIR)/lib -Wl,-rpath,$(abspath $(XSMM_DIR)/lib)
   $(libceeds) : LDLIBS += -lxsmm -ldl
   MKL ?= 0
-  ifneq (0,$(MKL))
-    BLAS_LIB = -Wl,--no-as-needed -lmkl_intel_lp64 -lmkl_sequential -lmkl_core -lpthread -lm -ldl
-  else
+  ifeq (0,$(MKL))
+  ifeq ($(MKLROOT),)
     BLAS_LIB = -lblas
+  else
+    BLAS_LIB = -Wl,--no-as-needed -lmkl_intel_lp64 -lmkl_sequential -lmkl_core -lpthread -lm -ldl
```

Comment:
Add -L and rpath flags.

Reply:
For MKL? We haven't needed those flags on CU's machine in the past.